ingest-app/src/cmr/ingest/services/jobs.clj

```diff
@@ -140,6 +141,21 @@
 ;; Only one node needs to refresh the cache because we're using the  fallback cache with Redis cache.
 ;; The value stored in Redis will be available to all the nodes.
 
+(defconfig email-server-host
+  "The host name for email server."
+  {:default "gsfc-relay.ndc.nasa.gov"
+   :type String})
+
+(defconfig email-server-port
+  "The port number for email server."
+  {:default 25
+   :type Long})
+
+(defconfig cmr-mail-sender
+  "The cmr email sender's email address."
+  {:default "cmr-support@earthdata.nasa.gov"
```

Comment:
The default value should be "". We don't want to put this value in the source code.